core/arch/arm/dts/fsl-lx2160a-qds.dts

```diff
@@ -21,6 +21,11 @@
 		stdout-path = "serial0:115200n8";
 	};
 
+	povdd {
+		povdd-gpio-controller = <0>;
+		povdd-gpio-pin = <26>;
```

Comment:
One last query regarding this, Did you tested this driver on both LX2160A-RDB and LX2160A-QDS boards ?

Reply:
No we don't have access to those boards, only the lx2160a. I had updated them with the 2160 dts but i realize now that wasn't necessary. I can go ahead and remove them

Reply:
Ok, Which board variant of LX2160a you are working on ?

Reply:
Sorry for the delay, we're using the LX2162a